autoload/vundle/installer.vim

```diff
@@ -393,6 +393,7 @@ func! s:make_sync_command(bang, bundle) abort
                   \ 'git fetch',
                   \ 'git reset --hard origin/HEAD',
```

Comment:
It's usual that the wrong codes "works", but this doesn't means it's right.

What if the `a:bundle.revision` branch uses a new submodule ?